src/Library/Plugin/Plugin.php

```diff
@@ -43,6 +43,15 @@ class Plugin {
 	 */
 	protected $file;
 
+	/**
+	 * Plugin specific config array.
+	 *
+	 * @since SINCEVERSION
+	 * @var array
+	 * @access protected
+	 */
+	protected $pluginConfig = [];
```

Comment:
Because this class represents a plugin, what do you think about changing `$pluginConfig` to simply `$configs`

Reply:
While the Plugin class does not have a $config or $configs property already, it does have a method called getConfig() which pulls config data from the src\library.global.php file, which is a global library config file. To specify that this is to pull the config data from the actual plugin, it named it $pluginConfig